Core/StitchCoreSDK/Sources/StitchCoreSDK/Internal/Net/AppMetadata.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+/// Application Metadata
+public final class AppMetadata: Codable, CustomStringConvertible {
```

Comment:
[nit] similar to my comment on the Android PR, this class could be named `APIAppMetadata` if we want to more closely follow naming conventions for things that represent JSON coming from the API

Reply:
fixed